### DISCUSSION TITLE
количество выводимых элементов (selectLength)

### DISCUSSION THREAD

**Discussion by IrinaG1090** [totumonline/totum-ru-issues-and-discussions#493] [Как это сделать?] [answered]
Привет, ребята! Подсветите пожалуйста.
есть поле селект.
в селект-коде вот такое:
=:SelectList(table: 'users';field: 'fio')

Самих ФИО много в таблице users. (70 шт)

Как воспользоваться selectLength чтоб, вывести в выпадающий список только определенные id (ФИО)? например, хочу вывести в выпадающий список селекта id 10, 11, 12, 13, 14, 38, 63. 
А остальные чтоб были спрятаны.

Это надо ручной селект делать или через  selectLength можн отразить? 
Если можно  - то как?
буду рада примерам конкретным

Заранее благодарю за любой подсвет

**Comment by Re-Requeste** [comment]
=:SelectList(table: 'users';field: 'fio';  where: 'id' =  json['10, 11, 12, 13, 14, 38, 63'])
Попробуйте через where

**Reply by IrinaG1090** [reply]
благодаю вас за энергию/ попробовала вот так  

```
=:SelectList(table: 'users'; field: 'fio'; where: 'field' =   json`[10, 11, 12, 13, 14, 38, 63]`)
```

не вышло(((((

**Reply by erpic-ru** [reply, marked as answer]
Если мы говорим про селект-код (а не просто код), то должно быть так: 
```
=: selectRowListForSelect(table: 'users';  field: 'fio';  where: 'id' = json`[10,12,15]`; order: 'fio' asc)

```

**Reply by Re-Requeste** [reply]
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/8895da76-12de-4a69-a887-71c893be91ca)
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/c310beb7-7070-4088-bf9d-fac8f83805ac)
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/5386cece-8d1d-4a0d-83f7-067ce16bff86)

1) json [' номера '] - указывается через ковычки
2) field поле, т.е. в моем примере исользуется id

**Reply by IrinaG1090** [reply]
круто сработалооооооооо!!!!!! пасибааааа!!!
